### PR TITLE
New version: Microsoft.PowerShell version 7.4.3.0

### LINKED ISSUE #158578: [Update Request]:  PowerShell
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

Microsoft.PowerShell

### Package Version

7.4.3

### Please describe the changes you would like to see

New release here: [v7.4.3 Release of PowerShell](https://github.com/PowerShell/PowerShell/releases/tag/v7.4.3)

### PR DESCRIPTION
**Pull request by Dragon1573**
## Checklist for Pull Requests

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] This PR related to #158578 but didn't solve it.

## Manifests

- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/158598)

### CODE REVIEW

File: manifests/m/Microsoft/PowerShell/7.4.3.0/Microsoft.PowerShell.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Microsoft.PowerShell
+PackageVersion: 7.4.3.0
```
Comment (Dragon1573) [inline manifests/m/Microsoft/PowerShell/7.4.3.0/Microsoft.PowerShell.installer.yaml:5]:
```suggestion
PackageVersion: 7.4.3.0
MinimumOSVersion: 10.0.17763.0
```

Suggested by @wingetbot

**Comment by stephengillie** [comment]
Hi @anamnavi,

Do you approve of these changes?

(Automated response - build 886.)

**Comment by stephengillie** [comment]
Hi @Dragon1573,

This manifest adds Apps and Features entries that aren't present in previous PR versions. These entries should be added to the previous versions, or removed from this version.

(Automated response - build 886.)

**Comment by anamnavi** [comment]
@stephengillie I'm putting a PR out for this, will link it in a few

**Comment by anamnavi** [comment]
@stephengillie please review this PR: https://github.com/microsoft/winget-pkgs/pull/158784

@Dragon1573 thanks for creating this, but the PowerShell team decided we will not include .msi installer for arm in our winget releases yet, but it will be available on our Github release page. Thanks!

**Comment by stephengillie** [comment]
Close with reason: Duplicate of #158784;

**Comment by MichaMican** [comment]
@anamnavi Are there at least any plans of bring the arm installer back to winget in the future? I mean Microsoft released the CP+ ARM Laptops as their new flagship - Tbh I don't fully understand why you guys are trying to withheld this from endusers, now that it will be relevant for as many people as ever till this point...
Does this imply Microsoft itself does not trust in the readyness of their own product or is it an issue with winget?
To my knowledge the ARM installer was even present for one of the past versions and was patched out.

**Pull request closed without merging**